packages/driver/src/cy/commands/cookies.ts

```diff
@@ -297,34 +281,31 @@ export default function (Commands, Cypress, cy, state, config) {
 
       options.timeout = options.timeout || config('defaultCommandTimeout')
 
-      const cookie = pickCookieProps(options)
+      const cookie = _.extend(pickCookieProps(options), { name, value })
+      let resultingCookie: Cypress.Cookie
+      let log: Log | undefined
 
       if (options.log) {
-        options._log = Cypress.log({
+        log = Cypress.log({
           message: [name, value],
           timeout: responseTimeout,
           consoleProps () {
-            let c
             const obj = {}
 
-            c = options.cookie
-
-            if (c) {
-              obj['Yielded'] = c
+            if (resultingCookie) {
+              obj['Yielded'] = resultingCookie
             }
 
             return obj
           },
         })
       }
 
-      const onFail = options._log
-
       cookie.sameSite = normalizeSameSite(cookie.sameSite)
 
       if (!_.isUndefined(cookie.sameSite) && !VALID_SAMESITE_VALUES.includes(cookie.sameSite)) {
         $errUtils.throwErrByPath('setCookie.invalid_samesite', {
-          onFail,
+          onFail: log,
```

Comment:
This continues the existing log, but do we need to pass the log to onFail? Wondering if the commandQueue's fail command logic logs these error correctly for the cookie log.

Reply:
This was the existing behavior, I just removed the part where it assigns `log` to the variable `onFail` beforehand, so there's no change in behavior.